package.json

```diff
@@ -18,42 +18,33 @@
   "main": "dist",
   "sideEffects": false,
   "scripts": {
-    "build": "yarn updateStyles && yarn updateSprite",
-    "bump": "yarn pre-bump && release-it",
-    "bump:minor": "yarn pre-bump && release-it minor",
+    "build": "yarn webpack:dev && yarn webpack:dist && yarn svg-sprite && yarn copy:sprite && yarn copy:styles",
     "bump:major": "yarn pre-bump && release-it major",
-    "copy:all": "yarn copy:dist && yarn copy:loch",
-    "copy:dist": "copyfiles --flat ./.lochness/dist/index.html ./dist/displayComponents.js ./dist/displayComponentStyles.css ./docs/",
-    "copy:loch": "copyfiles --flat .lochness/dist/assets/* ./docs/assets/",
-    "dev": "webpack --env=dev --watch",
-    "dist-watch": "webpack --env=dist --watch",
-    "dist": "webpack --env=dist",
-    "gh": "webpack --env=dev && yarn loch:build && yarn copy:all",
-    "lint-fix": "npm run lint -- --fix",
-    "lint": "eslint --cache --format=node_modules/eslint-formatter-pretty .",
-    "loch:build": "node ./node_modules/loch-ness/bin/build.js -e dev -c ./ -j src/defaults.json",
-    "loch:clean": "rimraf .lochness",
-    "pre-bump": "yarn build && git add . && git commit -m 'compiled styles and sprite'",
-    "serve": "loch -e dev -c ./ -j src/defaults.json",
-    "start": "yarn loch:build && concurrently --kill-others \"yarn dev\" \"yarn serve\"",
-    "svgSprite": "./.prep-icon.sh && ./.generate-sprite.sh",
-    "test:v:build": "yarn loch:build",
-    "test:v:buildFresh": "concurrently \"yarn build\" \"yarn test:v:build\"",
-    "test:v:clean": "yarn loch:clean",
-    "test:v:ref": "visualTest -e dev -c ./ -j src/defaults.json -t reference",
-    "test:v": "visualTest -e dev -c ./ -j src/defaults.json",
-    "test:visuals": "yarn test:v:clean && yarn build && yarn test:v:build && yarn test:v",
+    "bump:minor": "yarn pre-bump && release-it minor",
+    "bump": "yarn pre-bump && release-it",
+    "copy:gh-pages": "copyfiles --flat .lochness/dist/index.html dist/displayComponents.js dist/displayComponentStyles.css docs/ && copyfiles --flat .lochness/dist/assets/* docs/assets/",
+    "copy:sprite": "copyfiles --flat src/Icon/sprite.html dist/sprite.html",
+    "copy:styles": "copyfiles --flat dist/styles.css dist/styles.scss",
```

Comment:
I'm not sure, I guess you forgot to add the destination folder which in this case I think docs